version/1/4/fr/code_of_conduct.txt

```diff
@@ -0,0 +1,78 @@
+personnelle, la taille physique, la race, l'origine éthnique, l'âge, la
+religion ou la nationalité.
+
+Nos critères
+
+Exemples de comportements qui contribuent à créer un environnement positif :
+
+* l'utilisation d'un langage ouvert et accueillant
+* être respectueux des différents points de vues et expériences
+* accepter poliment les critiques constructives
+* se concentrer sur ce qui est meilleur pour la communauté
+* faire preuve d'empathie envers les autres membres de la communauté
+
+Exemples de comportements non acceptables :
+
+* l'utilisation de langage ou d'imagerie sexualisés et les avances sexuelles
+non bienvenues
+* le _trolling_, les commentaires insultants ou désobligeants, et les
+attaques personnelles ou politiques
+* le harcèlement en public ou en privé
+* la publication d'informations privées de tierces personnes, telles que
+  les adresses physiques ou électroniques, sans permission explicite
+* toute conduite qui pourrait être raisonnablement considérée comme inappropriée
+dans un milieu professionnel
+
+Nos responsabilités
+
+Les mainteneurs doivent clarifier les critères de comportement acceptables
+de ce projet : il est attendu d'eux qu'ils prennent les mesures correctives
+justes comme réponse à tous comportements inacceptables.
+
+Les mainteneurs du projet ont le droit et la responsabilité de supprimer,
+modifier ou rejeter les commentaires, _commits_, code, modifications du wiki,
+questions et autres contributions qui ne respectent pas ce Code de Conduite,
+ou de bannir temporairement ou définitivement tout contributeur pour des
+comportements qu'ils jugent inappropriés, menaçants, injurieux, ou nuisibles.
+
+Objectifs
+
+Ce Code de Conduite s'applique à la fois au sein des espaces de projets
+ainsi que dans les espaces publics quand un individu représente le projet
+ou sa communauté. Les exemples de représentations d'un projet ou d'une
+communauté incluent le fait d'utiliser une adresse email propre au projet,
+de poster sur les réseaux sociaux avec un compte officiel, ou d'intervenir
+comme représentant programmé à un événement en-ligne ou hors-ligne. La
+représentation du projet pourra être autrement définie et clarifiée par
+les mainteneurs du projet.
+
+Application
+
+Les cas de comportements abusifs, harcelants ou autrement inacceptables
+peuvent être signalés en contactant l'équipe du projet à
+[INSERER ADRESSE EMAIL]. Toutes les plaintes seront examinées et étudiées
+et se traduiront par une réponse jugée nécessaire et appropriée aux
+circonstances. L'équipe du projet s'engage à garder confidentielles les
+informations de la personne qui remonte un incident. Plus de détails sur
+la politique de mise en application des règles peuvent être publiés séparément.
+
+Les mainteneurs du projet qui ne suivent ou qui n'appliquent pas le Code de
+Conduite de bonne foi, s'exposent temporairement ou de façon permanente à des
+repercussions définies par d'autres membres extérieur à la direction du
+projet
```

Comment:
Ok so the last commit of @guilro is correct. Thanks @CoralineAda. I think we're good.